features/core-version.feature

```diff
@@ -16,4 +16,5 @@ Feature: Find version for WordPress install
       WordPress version: 4.4.2
       Database revision: 35700
       TinyMCE version:   4.208 (4208-20151113)
+      Package Language:  en_US
```

Comment:
Can we call this "Package language" (note the lowercase "language") for consistency with the other keys?